packages/components/app/styles/components/tabs.scss

```diff
@@ -6,60 +6,63 @@
 
 $indicator-height: 3px;
 
-.hds-tabs {}
+// .hds-tabs {}
 
 // Sub-components:
-// NOTE: hds-tab-list is still part of parent component may separate into child later as needed. Will consider renaming class then
-.hds-tab-list {
+// NOTE: hds-tabs__tablist-wrapper is still part of parent component, may separate into child later as needed. Will consider re-naming class then
+.hds-tabs__tablist-wrapper {
   padding-bottom: round(calc($indicator-height / 2));
   // overflow-x: auto;
   // -webkit-overflow-scrolling: touch;
   // white-space: nowrap; // move
 }
 
 // (Note: may rename later, this part of structure could still be changed as things evolve)
-.hds-tab-list__list {
-  border-bottom: 1px solid var(--token-color-border-primary);
-  list-style: none;
-  padding: 0;
-  margin: 0;
+.hds-tabs__tablist {
   display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  border-bottom: 1px solid var(--token-color-border-primary);
 }
 
 .hds-tabs__tab {
   position: relative; // for positioning temp indicator
 
-  // temporary indicator
-  /* TODO: refactor
-  &:first-child:after {
-    background: var(--token-color-foreground-action);
-    border-radius: 3px;
-    content: "";
-    display: block;
-    height: $indicator-height;
-    position: absolute;
-    left: 0;
-    right: 0;
-    bottom: round(calc($indicator-height / 2 * -1));
+  &.hds-tabs__tab--is-selected {
+    /* temporary indicator TODO: refactor WIP */
+    &::after {
+      position: absolute;
+      right: 0;
+      bottom: round(calc($indicator-height / 2 * -1));
+      left: 0;
+      display: block;
+      height: $indicator-height;
+      background: var(--token-color-foreground-action);
+      border-radius: 3px;
+      content: "";
+    }
+
+    > a {
+      color: var(--token-color-foreground-action);
+    }
   }
-  */
 
-  a {
+  > a {
```

Comment:
[suggestion] I think this needs a `white-space: nowrap`:

Without it: 
![CleanShot 2022-09-27 at 11 39 40](https://user-images.githubusercontent.com/4587451/192585469-f1254a93-689b-4a0c-9899-1d47561986c9.png)

After adding it: 
![CleanShot 2022-09-27 at 11 40 35](https://user-images.githubusercontent.com/4587451/192585646-9977f5c1-1e4c-4756-b0f3-167830e0e4e4.png)

Reply:
@MelSumner the tabs will have a dynamic overflow (see CRD) so no need for text wrapping (at least for now, unless we change our mind on the overflow behaviour); this will be implemented in follow-up tickets

Reply:
I was going to refine wrapping when I worked on the overflow navigation part but I'll go ahead and add that, thanks.